client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
+
+const EditReviewForm = (props) => {
+  const [editReview, setEditReview] = useState({
+    body: "",
+    score: "",
+    userId: "",
+    reviewId: "",
+    showId: ""
+  })
```

Comment:
userId, reviewId and showId shouldnt be stored in state. They will always be the same for a given EditReviewForm. Just take the ones passed as props.